### PR TITLE
[4.1] Align the inline help button to the rest of the btns

### PR DESCRIPTION
**Pull request by dgrammatiko**
Pull Request for Issue # .

### Summary of Changes
The static assets **should only be added inside a layout**. The inlinehelp toolbar button is adding the assets (a js file) inside the logic and this makes it impossible to override/remove the script in an easy fashion (eg by editing the relative layout). Also it's pretty obvious that this is wrong as it's the only button that doesn't adhere to the CMS' norm (add the assets only inside the layouts). 

### Testing Instructions
Apply the PR and check that the inlinehelp still works as before (check the global options)

### Expected result AFTER applying this Pull Request

No assets added in the logic
Screenshot 2022-03-23 at 14 09 19

All the assets added in the layouts
Screenshot 2022-03-23 at 14 08 47

### Documentation Changes Required

No, but it would be good to pull a flag whenever a PR is adding static assets in places other than the layouts

### CODE REVIEW

**Comment by laoneo** [comment]
I have tested this item :white_check_mark: successfully on 50c0643e2cc655c5fc0ed3cbfeac06a8014f7948<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37356">issues.joomla.org/tracker/joomla-cms/37356</a>.</sub>

File: libraries/src/Toolbar/ToolbarHelper.php [hunk 1]
```diff
@@ -191,23 +191,6 @@ public static function help($ref, $com = false, $override = null, $component = n
 	 */
 	public static function inlinehelp(string $class = "hide-aware-inline-help")
 	{
-		/** @var HtmlDocument $doc */
-		try
-		{
-			$doc = Factory::getApplication()->getDocument();
-
-			if (!($doc instanceof HtmlDocument))
```
Comment (Quy) [inline libraries/src/Toolbar/ToolbarHelper.php:199]: Please delete line13.

Comment (dgrammatiko) [reply libraries/src/Toolbar/ToolbarHelper.php:199]: done

File: libraries/src/Toolbar/ToolbarHelper.php [hunk 2]
```diff
@@ -191,23 +190,6 @@ public static function help($ref, $com = false, $override = null, $component = n
...
-			{
-				return;
-			}
-
-			$doc->getWebAssetManager()->useScript('inlinehelp');
-		}
-		catch (Throwable $e)
```
Comment (Quy) [inline libraries/src/Toolbar/ToolbarHelper.php:206]: Oops one more. Please delete line 18.

Comment (dgrammatiko) [reply libraries/src/Toolbar/ToolbarHelper.php:206]: ok

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 50c0643e2cc655c5fc0ed3cbfeac06a8014f7948<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37356">issues.joomla.org/tracker/joomla-cms/37356</a>.</sub>

**Comment by bembelimen** [comment]
Thx

**Pull request merged**